### PR TITLE
Efficios patch

### PR DESCRIPTION
**Pull request by Kerilk**

### CODE REVIEW

**Comment by TApplencourt** [comment]
no need to changing the bt2 patch? Do we also need to update the version of `babeltrace2` ?

**Comment by TApplencourt** [comment]
LGTM

File: packages/lttng-ust/package.py [hunk 1]
```diff
@@ -14,12 +14,15 @@ class LttngUst(AutotoolsPackage):
 
     maintainers = ['Kerilk']
 
-    version('2.13.2', sha256='08679a1dfc2c9428885273861628aa5d828972f389a38bb08fdce39adee589b7')
```
Comment (TApplencourt) [inline packages/lttng-ust/package.py:17]: should we keep 2.13.2?

Comment (Kerilk) [reply packages/lttng-ust/package.py:17]: No I don't know how to apply the patch there so for now we should get rid of it.

Comment (Kerilk) [reply packages/lttng-ust/package.py:17]: we could add from 2.13.4 on

**Review by TApplencourt — APPROVED** [review]

**Pull request merged**